quotes.py

```diff
@@ -0,0 +1,258 @@
+#    Copyright (C) 2019 Pavel (pqhaz)
+
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import logging
+import requests
+import ujson
+import base64
+
+from .. import loader, utils
+from PIL import Image
+from io import BytesIO
+from urllib.error import HTTPError
+from telethon.errors.rpcerrorlist import ChannelPrivateError
+from telethon.tl.functions.channels import GetParticipantRequest, GetFullChannelRequest
+from telethon.tl.functions.users import GetFullUserRequest
+from telethon.tl.types import (MessageEntityBold, MessageEntityItalic,
+                               MessageEntityMention, MessageEntityTextUrl,
+                               MessageEntityCode, MessageEntityMentionName,
+                               MessageEntityHashtag, MessageEntityCashtag,
+                               MessageEntityBotCommand, MessageEntityUrl,
+                               MessageEntityStrike, MessageEntityUnderline, Channel)
+
+logger = logging.getLogger(__name__)
+
+
+def register(cb):
+    cb(QuotesMod())
+
+
+class QuotesMod(loader.Module):
+    """Quote a message."""
+    def __init__(self):
+        self.config = loader.ModuleConfig("API_TOKEN", "", "API Key/Token for Quotes.",
+                                          "API_URL", "http://api.antiddos.systems", "API URL for Quotes.")
+        self.name = _("Quotes")
+
+    async def client_ready(self, client, db):
+        self.client = client
+
+    async def quotecmd(self, message):  # noqa: C901
+        """Quote a message.
+        Usage: .quote (template) (optional: file/force_file/sticker)
+        Sends as sticker by default
+        Needs an API Token."""
+
+        args = utils.get_args(message)
+        reply = await message.get_reply_message()
+
+        if not reply:
+            return await utils.answer(message, '<code>You didn\'t reply to a message.</code>')
+
+        if not args:
+            return await utils.answer(message, '<code>You didn\'t specify template</code>')
+
+        if reply.media:
+            msg = await message.edit('<code>Processing... Note: only message will get processed.</code>')
+        else:
+            msg = await message.edit('<code>Processing...</code>')
+
+        username = ''
+        no_pfp = True
+        colour = '#b48bf2'
+        admintitle = ''
+        username = ''
+        pfp = ''
```

Comment:
Ummmmmmm. Afaik you can't have a string for your photo. None is more appropriate.